src/DocumentFormat.OpenXml/ParaId/IParagraphIdHolder.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace DocumentFormat.OpenXml.Wordprocessing
+{
+    /// <summary>
+    /// Defines operations for elements holding w14:paraId (ParagraphId) values,
+    /// i.e., <see cref="Paragraph" /> and <see cref="TableRow" /> instances.
+    /// </summary>
+    public interface IParagraphIdHolder
```

Comment:
I generally am very cautious of exposing public surface changes unless I'm very happy with their shape. Once exposed, they will not be removed without being a breaking change. For me, the bar tends to be fairly high to actually expose something as public.

This feels like it could be made more generic and used elsewhere, but not in its current form. If there's a use case to make this public, we'll need to iterate on the design of it a bit more.